src/components/cms/MyDevices/MapContainer.js

```diff
@@ -115,13 +119,19 @@ class MapContainer extends Component {
             visible={showingInfoWindow}
           >
             <div>
-              <p>
-                {`Mac Address: ${selectedPlace.macId}`}
-                <br />
-                {`Room: ${selectedPlace.room}`}
-                <br />
-                {`Device Name: ${selectedPlace.title}`}
-              </p>
+              {selectedPlace.macId &&
```

Comment:
tooltipRender can be passed as a prop and its logic can be defined in the parent component.